config/default.yml

```diff
@@ -1138,6 +1138,13 @@ Rails/WhereNotWithMultipleConditions:
 Style/AndOr:
   EnforcedStyle: conditionals
 
+Style/FormatStringToken:
+  AllowedMethods:
+    - redirect
+  # Deprecated.
+  IgnoredMethods:
+    - redirect
```

Comment:
Why is redirect added to both `AllowedMethods`, and `IgnoredMethods`?  Seems contradictory, and makes it unclear what the end result would be from that config.  Ignored > Allowed?  Is that necessary?

Reply:
I replicated this config in my local:
```
Warning: obsolete parameter `IgnoredMethods` (for `Style/FormatStringToken`) found in .rubocop.yml
`IgnoredMethods` has been renamed to `AllowedMethods` and/or `AllowedPatterns`.
```

How does it make sense for `AllowedMethods` to have the same meaning as `IgnoredMethods`?  Shouldn't they be opposites?

Reply:
The [documentation](https://www.rubydoc.info/gems/rubocop/1.22.1/RuboCop/Cop/IgnoredMethods) for `IgnoredMethods` was:

> This module encapsulates the ability to ignore certain methods when parsing

The [documentation](https://msp-greg.github.io/rubocop/RuboCop/Cop/AllowedMethods.html) for `AllowedMethods` is the opposite:

> This module encapsulates the ability to allow certain methods when parsing.

How are these interchangeable?

Reply:
It seems that there is a lot of confusion around Allowed vs Ignored, and it has been made **worse** by making them identical.
https://github.com/rubocop/rubocop/pull/10829#issuecomment-1402871729